spec/spec_helper.rb

```diff
@@ -113,7 +113,7 @@
 Capybara.configure do |config|
   config.run_server = false
   config.default_driver = :poltergeist
-  config.save_path = __dir__ + "/../tmp"
+  config.save_path = ENV["CAPYBARA_SAVE_PATH"] or (__dir__ + "/../tmp")
```

Comment:
Found the official guidance if it's helpful :-) https://github.com/alphagov/styleguides/commit/8d732f5bc110dfc2dd822e3899775b42019c0227